ui/src/common/components/InlineList/InlineList.scss

```diff
@@ -16,10 +16,10 @@
   }
 
   $separator-padding: 0.25em;
-  @mixin separte-items-with($seperator, $padding-left: 0) {
-    // For every not empty `li`, add separator before all the not empty `li` that come after it
-    & > li:not(:empty) ~ li:not(:empty):before {
-      content: $seperator;
+  @mixin separte-items-with($separator, $padding-left: 0) {
+    // Add separator after all the non-empty `li` except the last one
```

Comment:
I don't think this works. What if we have a non-empty li then an empty li? It shouldn't add a separator between them since the next one is empty but it will add it in your case